requirements.txt

```diff
@@ -1,10 +1,13 @@
-pyyaml>=5.3.1
-numpy>=1.18.1
-numba==0.48.0  # Fix to librosa not respecting something deprecated
-librosa==0.7.2
 click>=7.1.1
+cytoolz>=0.10.1
+librosa==0.7.2
 lilcom>=1.1.0
+matplotlib~=3.2.2
+numba==0.48.0  # Fix to librosa not respecting something deprecated
+numpy>=1.18.1
+pandas>=1.0.3
+pytest~=5.4.3
+pyyaml>=5.3.1
+setuptools~=47.3.1
```

Comment:
I see there's still a couple of extra dependencies that shouldn't be here - can you revert the changes to this file altogether?

Reply:
Ah sorry, I just wanted to sort them and failed to check them carefully. I'll remove the unnecessary items.